app/src/main/java/br/org/cesar/discordtime/stickysessions/presentation/list/ListSessionsPresenter.java

```diff
@@ -58,7 +58,7 @@ public void onLoad() {
     }
 
     @Override
-    public void onPause() {
+    public void onStop() {
```

Comment:
We could also include a check for `mView != null` on Presenter's entry points.

Reply:
Sure. I'll add it.